### ISSUE TITLE
Technical: Docker implementation

### ISSUE THREAD

**Issue by emilyobvs** [DigitalWPI/digitalwpi#475]
Plan Docker implementation (TBD) - (ARC + WebDev + CL new task / new contract work?) - budget request in process for FY24; Derek is researching too including AI coding!

**Comment by emilyobvs** [comment]
Derek/Ermal are working with their team on the implementation, everything is in place but they are waiting on Anusha for the container.

**Comment by emilyobvs** [comment]
John has set up docker/container on Dev - set up as a service (initially we expected it to be set up on a new server altogether), Derek will follow up with Anusha to see if this set-up works for Cottage Labs

Note: Docker is only for the web application piece at this time

**Comment by emilyobvs** [comment]
Docker environment is set-up. Packages need to be installed on the server. We're running a new version of the operating system, set-up will be tested on it first.

Next steps: Derek will work with Anusha. He is also working with John and Mark in IT.

**Comment by emilyobvs** [comment]
IT set up new dedicated server for docker (nothing else on it). Derek sent login info to Anusha, waiting to hear back to see what else she needs.
